src/pages/partners/deriv-prime/index.tsx

```diff
@@ -1,13 +1,21 @@
 import React from 'react'
-import { WithIntl } from 'components/localization/withIntl'
 import Layout from 'components/layout/layout'
+import { WithIntl } from 'components/localization'
+import { SEO } from 'components/containers'
+import DerivPrime from 'features/pages/partners/deriv-prime'
+import PaymentAgentAffiliateNav from 'features/components/templates/navigation/payment-agent-nav'
 
-const DerivPrime = () => {
+const DerivPrimePage = () => {
     return (
-        <Layout type="partners" margin_top={10}>
-            <></>
+        <Layout type="noNav">
+            <SEO
+                description="_t_With Deriv Prime's liquidity solutions for online trading, you get unbeatable spreads on the commodities, cryptocurrencies, and forex markets._t_"
+                title="_t_Liquidity solutions provider | Deriv Prime_t_"
+            />
+            <PaymentAgentAffiliateNav />
+            <DerivPrime />
         </Layout>
```

Comment:
Implement this inside **src/features/pages/partners/deriv-prime/index.tsx** here. And Call that component here. For, reference you can check the pages we already did by new approach.